### PR TITLE
feat: add `math/base/special/asind`

### LINKED ISSUE #35: Implement `asind`
Compute the arcsine of `x`, where the output is in degrees. `@stdlib/math/base/special/asind`.

### PR DESCRIPTION
**Pull request by saisrikardumpeti**
Resolves #35 

## Description

> What is the purpose of this pull request?

This pull request:

-  Adds functionality to compute `asind` of any given input.
-  Say we have input `x`, so it should satisfies this `x <= 1` or `x >= -1` else it will return `NaN` 

| input | asind |
| -------- | ------- |
| 0  | 0    |
| 0.5 |  30   |
| $\frac{1}{\sqrt{2}}$ | 45    |
| $\frac{\sqrt{3}}{2}$ | 60 |
| 1 | 90 |

## Related Issues

> Does this pull request have any related issues?

This pull request:

-   resolves #35
-   fixes #

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

### CODE REVIEW

**Comment by saisrikardumpeti** [comment]
@Planeshifter this PR is ready for review.

File: lib/node_modules/@stdlib/math/base/special/asind/docs/repl.txt [hunk 1]
```diff
@@ -0,0 +1,28 @@
+
+{{alias}}( x )
+    Computes the arcsine of a double-precision floating-point number.
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/special/asind/docs/repl.txt]:
```suggestion
    Computes the arcsine (in degrees) of a double-precision floating-point
    number.
```

File: lib/node_modules/@stdlib/math/base/special/asind/docs/types/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+* var v = asind( 0.5 );
+* // returns ~30.0
+*
+* @example
+* var v = asind( Math.sqrt( 2 ) / 2 );
+* // returns ~45.0
+*
+* @example
+* var v = asind( Math.sqrt( 3 ) / 2 );
+* // returns ~60.0
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/special/asind/docs/types/index.d.ts]:
```suggestion
* var v = asind( Math.sqrt( 2.0 ) / 2.0 );
* // returns ~45.0
*
* @example
* var v = asind( Math.sqrt( 3.0 ) / 2.0 );
* // returns ~60.0
```

File: lib/node_modules/@stdlib/math/base/special/asind/lib/index.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+* // returns 0.0
+*
+* var v = asind( 0.5  );
+* // returns ~30.0
+*
+* var v = asind( Math.sqrt( 2 ) / 2 );
+* // returns ~45.0
+*
+* var v = asind( Math.sqrt( 3 ) / 2 );
+* // returns ~60.0
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/special/asind/lib/index.js]:
```suggestion
* var v = asind( Math.sqrt( 2.0 ) / 2.0 );
* // returns ~45.0
*
* var v = asind( Math.sqrt( 3.0 ) / 2.0 );
* // returns ~60.0
```

File: lib/node_modules/@stdlib/math/base/special/asind/lib/main.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+* var v = asind( 0.5  );
+* // returns ~30.0
+*
+* @example
+* var v = asind( Math.sqrt( 2 ) / 2 );
+* // returns ~45.0
+*
+* @example
+* var v = asind( Math.sqrt( 3 ) / 2 );
+* // returns ~60.0
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/special/asind/lib/main.js]:
```suggestion
* var v = asind( Math.sqrt( 2.0 ) / 2.0 );
* // returns ~45.0
*
* @example
* var v = asind( Math.sqrt( 3.0 ) / 2.0 );
* // returns ~60.0
```

File: lib/node_modules/@stdlib/math/base/special/asind/package.json [hunk 5]
```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@stdlib/math/base/special/asind",
+  "version": "0.0.0",
+  "description": "Compute the arcsine(in degrees) of a double-precision floating-point number.",
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/special/asind/package.json]:
```suggestion
  "description": "Compute the arcsine (in degrees) of a double-precision floating-point number.",
```

File: lib/node_modules/@stdlib/math/base/special/asind/package.json [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+    "url": "https://github.com/stdlib-js/stdlib/graphs/contributors"
+  },
+  "contributors": [
+    {
+      "name": "The Stdlib Authors",
+      "url": "https://github.com/stdlib-js/stdlib/graphs/contributors"
+    }
+  ],
+  "main": "./lib",
+  "gypfile": true,
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/special/asind/package.json]:
```suggestion
```

File: lib/node_modules/@stdlib/math/base/special/asind/README.md [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+// returns 0.0
+
+v = asind( 0.5 );
+// returns ~30.0
+
+v = asind( sqrt( 2 ) / 2 );
+// returns ~45.0
+
+v = asind( sqrt( 3 ) / 2 );
+// returns ~60.0
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/special/asind/README.md]:
```suggestion
v = asind( sqrt( 2.0 ) / 2.0 );
// returns ~45.0

v = asind( sqrt( 3.0 ) / 2.0 );
// returns ~60.0
```

**Review by Planeshifter — APPROVED** [review]

**Comment by Planeshifter** [comment]
Thank you; will merge once CI has passed.

**Pull request merged**